app/portainer/components/datatables/stacks-datatable/stacksDatatable.html

```diff
@@ -165,29 +183,31 @@
                   >{{ item.Name }}</a
                 >
                 <span ng-if="$ctrl.offlineMode">{{ item.Name }}</span>
-                <span ng-if="item.Regular && item.Status == 2" style="margin-left: 10px" class="label label-warning image-tag space-left">Inactive</span>
+                <span ng-if="item.Regular && item.Status == 2" class="label label-warning image-tag space-left">Inactive</span>
```

Comment:
on EE https://github.com/portainer/portainer-ee/pull/1849/files#diff-5973e760fdb6ae174b1e451d47ddf6eb660bbae295e84878338bd247b45a8989R194 we have `ml-2`. sync the files.

Reply:
also, ml-2 and space-left have the same functionality, decide between the two

Reply:
updated